src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFinder.java

```diff
@@ -95,6 +117,23 @@ public void onSuccess(Entity result) {
 		}
 	}
 	
+	private void multiSelectionOkClicked() {
+		//check for valid selection
+		if (view.getMultiEntityText().isEmpty()) {
+			synAlert.showError(DisplayConstants.PLEASE_MAKE_SELECTION);
+		} else {
+			lookupMultiEntity(view.getMultiEntityText(), new AsyncCallback<PaginatedResults<EntityHeader>>() {
```

Comment:
Please verify that selecting a single Synapse entity (when multi is configured) properly calls back with a list containing the single item.